clients/drcachesim/common/options.cpp

```diff
@@ -247,13 +247,15 @@ droption_t<std::string>
                           "Specifies the replacement policy for TLBs. "
                           "Supported policies: LFU (Least Frequently Used).");
 
-droption_t<std::string>
-    op_simulator_type(DROPTION_SCOPE_FRONTEND, "simulator_type", CPU_CACHE,
-                      "Simulator type (" CPU_CACHE ", " TLB ", " REUSE_DIST
-                      ", " REUSE_TIME ", " HISTOGRAM ", or " BASIC_COUNTS ").",
-                      "Specifies the type of the simulator. "
-                      "Supported types: " CPU_CACHE ", " TLB ", " REUSE_DIST
-                      ", " REUSE_TIME ", " HISTOGRAM "or " BASIC_COUNTS ".");
+droption_t<std::string> op_simulator_type(DROPTION_SCOPE_FRONTEND, "simulator_type",
```

Comment:
Please add to the op_LL_miss_file docs that for the miss analyzer its results will be written there

Reply:
Done.